src/components/CreateMessageSelect.tsx

```diff
@@ -126,60 +125,61 @@ export const CreateMessageSelect = () => {
                 <span class="material-symbols-outlined">close</span>
               </button>
             </li>
+            */}
           </ul>
         </div>
         <div class={wordListContainerStyle}>
           <div>
             <form action="">
-              <input type="text" />
-              <button type="submit">
+              <input type="text" value ="" id="kennsaku" placeholder="ここに検索したい語句を入力してください" oninput="kennsaku_show()"/>
                 <span class="material-symbols-outlined">search</span>
```

Comment:
```suggestion
              <button type="submit">
                <span class="material-symbols-outlined">search</span>
              </button>
```
スタイルがずれているので修正お願いします

ボタンを押すと検索リセットされるので、後でアイコン修正するかもしれない